packages/apps/composer-app/vite.config.ts

```diff
@@ -174,6 +174,9 @@ export default defineConfig({
       },
       authToken: process.env.SENTRY_RELEASE_AUTH_TOKEN,
       disable: process.env.DX_ENVIRONMENT !== 'production',
+      release: {
+        name: process.env.npm_package_version,
```

Comment:
how does this env get set?

Reply:
https://docs.npmjs.com/cli/v8/using-npm/scripts#packagejson-vars
npm (and pnpm as well) sets it

Reply:
🤯 

excellent